File: 151002/Redkovskiy/StoryTeller/publisher-service/src/main/java/com/example/publisherservice/dto/responseDto/StoryResponseTo.java
package com.example.publisherservice.dto.responseDto;

import lombok.Data;

import java.io.Serializable;

@Data
public class StoryResponseTo implements Serializable {

    private Long id;

    private String title;

    private String content;

    private String created;

    private String modified;

    private Long creatorId;
}

File: 251003/Kopytok/rv_lab_1/DTO/responces/StoryResponseTo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DTO.responces
{
    public class StoryResponseTo
    {
        public long Id { get; set; }
        public long EditorId { get; set; }
        public string Title { get; set; } = string.Empty;
        public string Content { get; set; } = string.Empty;
        public DateTime Created { get; set; }
        public DateTime Modified { get; set; }
    }
}
